modules/readJSON.js

```diff
@@ -2,6 +2,13 @@ const CryptoJS = require('crypto-js')
 const fs = require('fs')
 
 exports.readJSON = function() {
+	class userTz {
+		constructor(id, ianaTz) {
+	  	this.id = id;
+	  	this.ianaTz = ianaTz;
+		}
+	}
```

Comment:
why do we need a class for this?

Reply:
Because it is **significantly** easier to work with a class which has two attributes than with a key-value pair in a dictionary. For one, I can sort the array of classes.

Reply:
I'll add a commit with the sort now, so you can see why.

Reply:
I could also make it an array of dicts, of course; what I really need is a dataclass.

Reply:
changed in #6ee575e; used object instead of class